example/README.md

```diff
@@ -0,0 +1,3 @@
+# Flame Tiled example
+
+description: Simple project to showcase the usage of flame_tiled
```

Comment:
I don't think we need `description:` on a markdown right?

Reply:
Nope, gonna remove it